src/consumer/project.rs

```diff
@@ -24,16 +24,16 @@ pub struct MetricsResponse {
 
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
 pub enum ProjectMetricMetricType {
-    #[serde(rename = "unknown")]
+    #[serde(rename = "UNKNOWN")]
     #[default]
     UNKNOWN,
-    #[serde(rename = "user_count")]
+    #[serde(rename = "USER_COUNT")]
     USERCOUNT,
```

Comment:
We *probably* want this to be a variant like `UserCount`, but since that would be a breaking change, I guess we have to live with it for now.